packages/react-native/project.json

```diff
@@ -14,9 +14,7 @@
     },
     "TODOrelease": {
```

Comment:
Would be nice to not have this be TODO and maybe wire it up to the release action that fz added for electron

Reply:
Yeah, agreed. Might take a look at that this week.